core/drivers/crypto/caam/caam_ctrl.c

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/**
+ * @copyright 2017-2019 NXP
+ *
+ * @file    caam_ctrl.c
+ *
+ * @brief   CAAM Global Controller.\n
+ */
+
+/* Global includes */
```

Comment:
Agreed and I also do not like the separation `/* Global includes */` vs. `/* Local includes */`. We don't normally do that, and if we accept this here, this will undoubtedly be used as a template by other contributors.

Reply:
ok, I'll remove this split that was helpful for me.